app/components/system_header/index.tsx

```diff
@@ -39,18 +40,24 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
         header: {
             flex: 1,
             flexDirection: 'row',
+            marginTop: 10,
```

Comment:
Could you help me understand why the `marginTop` addition is necessary?

Reply:
As mentioned in the ticket, the layout for system ephemeral post not placed as expected. But this change affects the other posts, will fix by adding condition specific to ephemeral post.